pkgs/development/python-modules/openapi-codec/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+
+, coverage
+, pytest
+, flake8
+# runtime
+, coreapi
+}:
+
+buildPythonPackage rec {
+  pname = "openapi-codec";
+  version = "1.3.2";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "core-api";
+    repo = "python-openapi-codec";
+    rev = version;
+    hash = "sha256-2pRt/AuNofj2k9FaO5JNBt4fFZwtZyvgalyPp0u70EY=";
+  };
+
+  propagatedBuildInputs = [
+    coreapi
+  ];
+
+  nativeCheckInputs = [
+    coverage
```

Comment:
Can you try to remove coverage test with a patch to setup.py or similar? We usually skip those

Reply:
It's not trivial to rip out the coverage with a small patch, as https://github.com/core-api/python-openapi-codec/blob/master/runtests uses it a bunch. I can patch it out, but it's a relatively big patch. Should I do so?

Reply:
@onny how should I proceed?

Reply:
We don't want to run coverage or flake8